src/MapfileStyleParser.ts

```diff
@@ -362,12 +361,18 @@ export class MapfileStyleParser implements StyleParser {
       markSymbolizer.visibility = false;
     }
 
-    // markSymbolizer.fillOpacity = TODO from symbol?;
+    if (mapfileStyle.symbol.filled) {
+      markSymbolizer.fillOpacity = mapfileStyle.symbol.filled.toLowerCase() === 'true' ? 1 : 0;
+    } else {
+      markSymbolizer.fillOpacity = 0;
+    }
 
     if (mapfileStyle.size) {
       markSymbolizer.radius = mapfileStyle.size / 2;
     }
 
+    markSymbolizer.rotate = mapfileStyle.angle ? mapfileStyle.angle * 1 : 0;
```

Comment:
What is the benefit of multiplying the angle by 1?

Reply:
`ANGLE`s that are specified in MapFiles with `AUTO`, `AUTO2` or `FOLLOW` instead of an explicit value are returned with `NaN` at this point. Is this considered? The `ANGLE` of `LABEL` allow `[double|auto|auto2|follow|attribute]`

Multiplying by 1, if an explicit value is present as a string, would generate a floating point number from it in any case.